src/expr/node.h

```diff
@@ -1539,6 +1539,10 @@ bool NodeTemplate<ref_count>::hasSubterm(NodeTemplate<false> t, bool strict) con
 
   for (unsigned i = 0; i < toProcess.size(); ++ i) {
     TNode current = toProcess[i];
+    if (current.hasOperator() && current.getOperator() == t)
```

Comment:
Add the HO flag since this test only makes sense to HO and both ``hasOperator`` and ``getOperator`` do switches

Reply:
I think the precedent here is to always check.  See:
https://github.com/CVC4/CVC4/blob/master/src/expr/node.cpp#L129
Basically, node is a generic utility that we probably don't want to depend on an theory/uf option.